packages/cozy-harvest-lib/src/components/infos/TriggerErrorInfo.jsx

```diff
@@ -67,14 +67,16 @@ export class TriggerErrorInfo extends PureComponent {
   }
 
   render() {
-    const { error } = this.props
+    const { className, error, ...rest } = this.props
```

Comment:
What can be in `...rest` ? If spreading props can be avoided we should

Reply:
You're right, we can get rid of it here, at least for now.